projects/plugins/crm/modules/woo-sync/includes/class-woo-sync.php

```diff
@@ -265,11 +265,11 @@ public function get_default_status_for_order_obj( $obj_type_id, $order_status )
 			);
 
 			if ( in_array( $order_status, $paid_statuses, true ) ) {
-				$status = __( 'Paid', 'zero-bs-crm' );
+				$status = 'Paid';
 			} elseif ( $order_status === 'checkout-draft' ) {
-				$status = __( 'Draft', 'zero-bs-crm' );
+				$status = 'Draft';
 			} else {
-				$status = __( 'Unpaid', 'zero-bs-crm' );
+				$status = 'Unpaid';
 			}
```

Comment:
Don't we need to add some kind of migration to be backwards compatible?

Reply:
We have this problem throughout the whole codebase.
E.g.:
```
includes/ZeroBSCRM.DAL3.Helpers.php

		// retrieves statuses from object :)
		function zeroBSCRM_getInvoicesStatuses(){
		    
		    // for DAL3+ these are hard typed, probably need to sit in the obj:
		    return array(
		    	__( 'Draft', 'zero-bs-crm' ),
		    	__( 'Unpaid', 'zero-bs-crm' ),
		    	__( 'Paid', 'zero-bs-crm' ),
		    	__( 'Overdue', 'zero-bs-crm' ),
		    	__( 'Deleted', 'zero-bs-crm' )
		    );
		}
```

Reply:
> Don't we need to add some kind of migration to be backwards-compatible?

I mentioned this briefly above; to my knowledge, only WooSync had this bug. A migration gets messy because we have know way of knowing if:

1. The user has changed languages since, or
2. The value in the database is not something they manually added.

Since there's an easy workaround (Woo resync), I think avoiding a migration is best.

Reply:
> We have this problem throughout the whole codebase.

Indeed, my goal here was primarily to address the specific Woo situation, and to make output more i18n-friendly. But I'll poke around and see if there are other cases we can clean things up.

Reply:
Fixed code block that was meant to set an invoice to "Unpaid" if it was "Draft" and emailed to a user: 56380a3f3d

Because `'Draft' !== __( 'Draft', 'zero-bs-crm' )` if the status has translations, the condition was never met previously.

Reply:
Fixed invoice bulk action status change which hasn't worked previously on non-English installs: ab85f34521

Reply:
Fixed invoice bulk action status change which hasn't worked previously on non-English installs: ab85f34521

Reply:
WooSync status mapping was previously using translated status, but now correctly uses the English status with translated output: 6ebf0c04f4